spyderlib/config/base.py

```diff
@@ -196,10 +196,6 @@ def add_image_path(path):
 
 add_image_path(get_module_data_path('spyderlib', relpath='images'))
 
-from spyderlib.otherplugins import PLUGIN_PATH
-if PLUGIN_PATH is not None:
-    add_image_path(osp.join(PLUGIN_PATH, 'images'))
-

```

Comment:
That is from @Nodd

I guess because now the plugins path is not a fixed thing but can be located in many places (namespace packages)